packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -452,84 +457,86 @@ export const UIModeView: React.FC<{}> = ({
-          {!isRunningTest && progress && <div data-testid='status-line' className='status-line'>
-            <div>{progress.passed}/{progress.total} passed ({(progress.passed / progress.total) * 100 | 0}%)</div>
-          </div>}
-          {isRunningTest && progress && <div data-testid='status-line' className='status-line'>
-            <div>Running {progress.passed}/{runningState.testIds.size} passed ({(progress.passed / runningState.testIds.size) * 100 | 0}%)</div>
-          </div>}
-          <ToolbarButton icon='play' title='Run all — F5' onClick={() => runTests('bounce-if-busy', visibleTestIds)} disabled={isRunningTest || isLoading}></ToolbarButton>
-          <ToolbarButton icon='debug-stop' title={'Stop — ' + (isMac ? '⇧F5' : 'Shift + F5')} onClick={() => testServerConnection?.stopTests({})} disabled={!isRunningTest || isLoading}></ToolbarButton>
-          <ToolbarButton icon='eye' title='Watch all' toggled={watchAll} onClick={() => {
-            setWatchedTreeIds({ value: new Set() });
-            setWatchAll(!watchAll);
-          }}></ToolbarButton>
-          <ToolbarButton icon='collapse-all' title='Collapse all' onClick={() => {
-            setCollapseAllCount(collapseAllCount + 1);
-          }} />
-        </Toolbar>
-        <TestListView
-          filterText={filterText}
-          testModel={testModel}
-          testTree={testTree}
-          testServerConnection={testServerConnection}
-          runningState={runningState}
-          runTests={runTests}
-          onItemSelected={setSelectedItem}
-          watchAll={watchAll}
-          watchedTreeIds={watchedTreeIds}
-          setWatchedTreeIds={setWatchedTreeIds}
-          isLoading={isLoading}
-          requestedCollapseAllCount={collapseAllCount}
-          setFilterText={setFilterText}
-          onRevealSource={onRevealSource}
-        />
-        <Toolbar noShadow={true} noMinHeight={true} className='settings-toolbar' onClick={() => setTestingOptionsVisible(!testingOptionsVisible)}>
-          <span
-            className={`codicon codicon-${testingOptionsVisible ? 'chevron-down' : 'chevron-right'}`}
-            style={{ marginLeft: 5 }}
-            title={testingOptionsVisible ? 'Hide Testing Options' : 'Show Testing Options'}
-          />
-          <div className='section-title'>Testing Options</div>
-        </Toolbar>
-        {testingOptionsVisible && <SettingsView settings={[
-          singleWorkerSetting,
-          showBrowserSetting,
-          updateSnapshotsSetting,
-        ]} />}
-        <Toolbar noShadow={true} noMinHeight={true} className='settings-toolbar' onClick={() => setSettingsVisible(!settingsVisible)}>
-          <span
-            className={`codicon codicon-${settingsVisible ? 'chevron-down' : 'chevron-right'}`}
-            style={{ marginLeft: 5 }}
-            title={settingsVisible ? 'Hide Settings' : 'Show Settings'}
+      </div>}
+      sidebar={
+        <div className='vbox ui-mode-sidebar'>
```

Comment:
nit: if you merge this line with the previous one, the diff would be much much smaller because of no indent changes 😄